app/views/users/_organization.html.erb

```diff
@@ -1,5 +1,6 @@
 <% if @organization_membership %>
   <%= render "users/org_#{@organization_membership.type_of_user}" %>
+  <%# "org_admin", "org_guest", "org_member", etc. For all types, see app/models/organization_membership.rb  %>
```

Comment:
small nitpick: can we put the comment above the line that it refers to? that seems to be what we do elsewhere. for a second i didn't even realize that this was referencing the line above and thought it was just accidentally-commented-out code 🙈

Reply:
Aha yep! I _knew_ shouldn't have broken convention here 😅